### PR TITLE
Hw26.2

### PR DESCRIPTION
**Pull request by OleksiiTromsa**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String productIdString = req.getParameter("id");
+        Long productId = Long.valueOf(productIdString);
+        shoppingCartService.deleteProduct(shoppingCartService.getByUser(USER_ID),
+                productService.get(productId));
+
```
Comment (KostiantynVorobiov) [inline src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java]: Delete empty line

**Review by KostiantynVorobiov — APPROVED** [review]
Delete empty line and everything ok!

File: src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java (hunk 1 above)
Comment (OleksiiTromsa) [reply src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java]: Done

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

**Pull request merged**